src/components/ProductList/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function ProductList({ values }) {
+  return (
+    <>
+      {values && (
+        <div>
+          {values.docs.map((doc) => (
+            <p key={doc.id}>{doc.data().item}</p>
+          ))}
+        </div>
+      )}
+    </>
```

Comment:
Let's consider a more idiomatic way of doing this kind of components (where you either render some JSX or nothing).

Moving forward, I suggest you using ternary operators:

```jsx
function TheComponent() {
  return values ? <div>...</div> : null
}
```

With that, you don't have to use the `<>` anymore.

Reply:
Oh! It's awesome! It's easier! 👍 Thanks!